src/assets/i18n/en.json

```diff
@@ -318,7 +318,14 @@
       "speedCheckEmergency": "Speed Requirement - Emergency Stop",
       "speedCheckAvoidance": "Speed Requirement - Avoidance",
       "safetyAndBalanceQuestions": "Safety and balance questions",
-      "pcvDoorExercise": "PCV Door Exercise"
+      "pcvDoorExercise": "PCV Door Exercise",
+      "controlsFrontBrake": "Controls - Front brake",
+      "controlsRearBrake": "Controls - Rear brake",
+      "controlsThrottle": "Controls - Throttle",
+      "controlsBalanceSlowControl": "Controls - Balance / Slow control",
+      "positioningNormalRiding": "Positioning - Normal riding",
+      "bends": "Bends"
```

Comment:
Added to en.json -> should be welsh equivalents added to cy.json (unless you dont have them, in which case placeholders with [cy] prefixes should be added.

Reply:
OK will do. What would the placeholder look like if you can give me an example?

Reply:
Updated